### PR TITLE
Introducing a skeleton of metrics support in Stackdriver Exporter

### PR DESCRIPTION
**Pull request by simonz130**
- Added metrics support for Stackdriver Exporter
- Currently we don't support all types of monitored resources
- Lack substantial testing (in progress)

### CODE REVIEW

File: src/OpenCensus.Exporter.Stackdriver/Implementation/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    using System;
+
+    internal class Constants
+    {
+        public const string LABEL_DESCRIPTION = "OpenCensus TagKey";
+        public const string OPENCENSUS_TASK = "opencensus_task";
+        public const string OPENCENSUS_TASK_DESCRIPTION = "Opencensus task identifier";
+        public const string GCP_GKE_CONTAINER = "k8s_container";
+        public const string GCP_GCE_INSTANCE = "gce_instance";
+        public const string AWS_EC2_INSTANCE = "aws_ec2_instance";
```
Comment (SergeyKanzhelev) [inline src/OpenCensus.Exporter.Stackdriver/Implementation/Constants.cs]: I don't see you using those. Is it copy/paste from somewhere?

File: src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverMetricsExporterWorker.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                        Thread.Sleep(cancellationInterval);
+                        sleepTime = sleepTime.Subtract(cancellationInterval);
+                    }
+
+                    Thread.Sleep(sleepTime);
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
```
Comment (SergeyKanzhelev) [inline src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverMetricsExporterWorker.cs]: yeah... we will need to clean up error reporting

**Comment by SergeyKanzhelev** [comment]
@simonz130 do you want me to merge it?

File: src/OpenCensus.Exporter.Stackdriver/Implementation/Constants.cs (hunk 1 above)
Comment (simonz130) [reply src/OpenCensus.Exporter.Stackdriver/Implementation/Constants.cs]: This is from java exporter. 
Stackdriver names (prefix) metrics based on the environment in which they are collected. I am going to use these later on in the implementation. (future PRs).

File: src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverMetricsExporterWorker.cs (hunk 2 above)
Comment (simonz130) [reply src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverMetricsExporterWorker.cs]: Yeah, we should decide on how to log in OpenCensus library and apply it to exporters too.

**Comment by simonz130** [comment]
I will update this branch with main branch changes and then you can merge it.

**Pull request merged**